src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private String color;
+
+    public abstract int getArea();
+
+    public abstract void draw();
```

Comment:
It is not necessary to declare this method here because it already exists in Drawable interface